src/Marten/Events/Daemon/ProjectionDaemon.cs

```diff
@@ -139,7 +139,7 @@ public async Task StartShard(AsyncProjectionShard shard, CancellationToken cance
             await TryAction(parameters).ConfigureAwait(false);
         }
 
-        public async Task StopShard(string shardName, Exception ex = null)
+        public async Task StopShard(string shardName, Exception? ex = null)
```

Comment:
This is possibly a breaking change(?)

Reply:
NRT annotations aren't a breaking change as they only add compile-time metadata.

Reply:
Gotcha 👍